content/source/docs/cloud/architectural-details/security-model.html.md

```diff
@@ -38,7 +38,7 @@ Terraform Cloud executes Terraform configuration from connected VCS repositories
 
 ## Authorization Model
 
-<a href="/../../../assets/images/docs/terraform-cloud-authorization.png" target="_blank">
+<a href="../../../assets/images/docs/terraform-cloud-authorization.png" target="_blank">
 ![Terraform Cloud authorization model diagram](../../../assets/images/docs/terraform-cloud-authorization.png)
 </a>
```

Comment:
Let's try this, since it's the markdown way to do it 

```suggestion
[![Terraform Cloud authorization model diagram](../../../assets/images/docs/terraform-cloud-authorization.png)](../../../assets/images/docs/terraform-cloud-authorization.png)
```